integration-cli/docker_cli_history_test.go

```diff
@@ -121,7 +112,7 @@ func (s *DockerSuite) TestHistoryHumanOptionTrue(c *check.C) {
 	out, _ := dockerCmd(c, "history", "--human=true", "busybox")
 	lines := strings.Split(out, "\n")
 	sizeColumnRegex, _ := regexp.Compile("SIZE +")
-	humanSizeRegex, _ := regexp.Compile("^\\d+.*B$") // Matches human sizes like 10 MB, 3.2 KB, etc
+	humanSizeRegexRaw := "^\\d+.*B$" // Matches human sizes like 10 MB, 3.2 KB, etc
```

Comment:
Thanks for pointing that out.   I removed them, since even though it works, it is confusing.